src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -201,6 +201,17 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
                     HelpMessage = "Custom routes AddressPool specified by customer")]
         public string[] CustomRoute { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The NatRules for Virtual network gateway.")]
+        public PSVirtualNetworkGatewayNatRule[] NatRule { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Flag to enable and disable BgpRouteTranslationForNat on this VirtualNetworkGateway.")]
```

Comment:
I believe its help message should be updated also.

Reply:
@BethanyZhou I have updated help message as per your suggestion. Thanks